stablehlo/reference/ProcessGrid.h

```diff
@@ -0,0 +1,125 @@
+/* Copyright 2023 The StableHLO Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef STABLEHLO_REFERENCE_PROCESSGRID_H_
+#define STABLEHLO_REFERENCE_PROCESSGRID_H_
+
+#include <condition_variable>
+#include <cstdint>
+#include <map>
+#include <mutex>
+#include <utility>
+
+#include "mlir/IR/BuiltinTypes.h"
+#include "mlir/Support/LLVM.h"
+#include "stablehlo/reference/Tensor.h"
+
+namespace mlir {
+namespace stablehlo {
+
+// Object to represent StableHLO `process_id`.
```

Comment:
Simplify to "StableHLO `process_id`", similarly to "StableHLO `replica_id`" below? (Same for the comment on `ProcessGroup`).